bundle/deploy/terraform/tfdyn/convert_database_instance.go

```diff
@@ -0,0 +1,49 @@
+package tfdyn
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/cli/bundle/internal/tf/schema"
+	"github.com/databricks/cli/libs/dyn"
+	"github.com/databricks/cli/libs/dyn/convert"
+	"github.com/databricks/cli/libs/log"
+	"github.com/databricks/databricks-sdk-go/service/database"
+)
+
+type databaseInstanceConverter struct{}
+
+func (d databaseInstanceConverter) Convert(ctx context.Context, key string, vin dyn.Value, out *schema.Resources) error {
+	// Normalize the output value to the target schema.
+	vout, diags := convert.Normalize(database.DatabaseInstance{}, vin)
```

Comment:
I assume that when a new TF provider is available, you will use the schema.DatabaseInstance instead, and then you don't need the conversion for purge_on_delete below?